packages/date-time/src/components/Calendar/Calendar.types.ts

```diff
@@ -237,7 +237,7 @@ export interface ICalendarStrings {
   closeButtonAriaLabel?: string;
 
   /**
-   * Aria-label format string for the week number header. Should have 1 string param e.g. "week number {0}"
+   * Aria-label format string for the week number header. Should have 1 string param e.g. "week number 0"
```

Comment:
Not sure what the appropriate comment here should be. It's confusing to remove the curly braces around the replacement token identifier. It'a also annoying because (I think) the api documentation tool complains about invalid characters :(

Reply:
yeah, that's why i had to remove them.  @natalieethell do you know how we can use curley braces in our API docs? Seems API extractor hates them.

Reply:
You should be able to use \`{0}\` (with back-ticks)